src/google_map.js

```diff
@@ -364,6 +369,16 @@ export default class GoogleMap extends Component {
     return minZoom;
   }
 
+  _mapDomResizeCallback = () => {
+    this._setViewSize();
+    this._onBoundsChanged();
+    this.resetSizeOnIdle_ = true;
```

Comment:
The final solution is in the main thread below I will repeat it here 

```javascript
  _mapDomResizeCallback = () => {
    this.resetSizeOnIdle_ = true;
    if (this.maps_) {
      this.maps_.event.trigger(this.map_, 'resize');
    }
  }
```

Looks like it solves an issues I see.